openapi/openapi.yaml

```
@@ -478,6 +478,9 @@ tags:
       Use these operations to manage password resets.
       When a password reset is requested, an email with a token is sent.
       In the email the user must click a reset link, and then create a new password.
+  - name: Risk Score
+    description: |-
+      Use Risk score to flexibly configure Blocklists based on transaction Risk Factors.
```

issue: sentence case

I also think we should remove 'flexibly' as it does not add meaning to the description

```suggestion
      Use risk score operations to configure blocklists based on transaction risk factors.
```